tests/default-ceph-config.json

```diff
@@ -3,7 +3,7 @@
     "storageClassWindows":     "rook-ceph-block",
     "storageRWXFileSystem":    "nfs-csi",
     "storageRWXBlock":         "rook-ceph-block",
-    "storageRWOFileSystem":    "local",
```

Comment:
Change it to what?

Reply:
I changed it to rook-ceph-block

Reply:
Gotcha, missed that. I think it is fine in here. We still have the non rook ceph to test for WFFC type storage.

Reply:
I was just looking through some test runs and this surprised me,
For example, I'm pretty sure we would have liked fstrim tests for instance to be executed against ceph

Reply:
Yeah, IIRC we left it at local because that would test the WFFC portions. But I have no issues changing it to rook-ceph-block for the ceph lane, but we should have a non ceph lane as well.

Reply:
oh ok, that is a good reason. perhaps we should get a rook-ceph-block-wffc sc variation and use that then.